rca/project_styleguide/templates/patterns/pages/student/student_detail.html

```diff
@@ -0,0 +1,162 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags wagtailsettings_tags static %}
+{% get_settings %}
+
+{% block body_class %}
+    app--staff-detail theme-light
+{% endblock %}
+
+{% block title %}
+    {% if page.staff_title %}{{ page.staff_title }}{% endif %} {% if page.first_name %}{{ page.first_name }}{% endif %} {% if page.last_name %}{{ page.last_name }}{% endif %}
+{% endblock %}
+
+{% block content %}
+
+    <div class="page page--no-hero" typeof="schema:Person">
+        <header class="page__header bg bg--dark">
+            <div class="title-area title-area--staff-detail grid">
+                {% if page.profile_image %}
+                    <div class="title-area__image">
+                        {% include "patterns/atoms/profile-image/profile-image.html" %}
+                    </div>
+                {% endif %}
+                <div class="title-area__content">
+                    <nav class="title-area__breadcrumb" aria-label="Breadcrumb">
+                        {% include "patterns/molecules/breadcrumb/breadcrumb.html" %}
+                    </nav>
+                    {% if page.staff_title or page.first_name or last_name %}
```

Comment:
`or page.last_name`

Reply:
good spot, shall fix in the staff template too!